runners/console/index.ts

```diff
@@ -347,6 +347,22 @@ export class Console extends Runner {
         return result;
     }
 
+    runCreateNgProject(runCommand: RunCommand): RunResult {
```

Comment:
I think we should name the command 'createDevon4ngProject' to keep it consistent with createDevon4jProject. We can then use the devon syntax to create the project and run 'devon ng create ...'. It is the same as 'devon ng new', but on the devon website this is the way it is used.
@denise-khuu What do you think?

Reply:
I would change the name to 'createDevon4ngProject' but still give the option to create an angular project on it's own. We have a focus on devon4ng tutorials, but this also implies that we would have to use restoreDevonfwIde in all angular tutorials, which will take a lot of time to setup.

Reply:
Do you mean to name it createDevon4ngProject but use the 'normal' way (ng new) when no ide was installed before? Or to implement a own function for this?

Reply:
For now I changed the function to createDevon4ngProject.

Reply:
Yes, using the normal way, if the IDE is not installed, is what I meant @GuentherJulian